hs_model_program/tests/test_model_program_metadata.py

```diff
@@ -68,7 +75,7 @@ def test_allowed_file_types(self):
         self.assertEquals(self.resModelProgram.metadata.program, None)
 
         # Upload any other file type should pass both the file pre add check post add check
-        files = [UploadedFile(file=self.text_file_obj, name=self.text_file_obj.name)]
+        files = [UploadedFile(file=self.text_file_obj_2, name=self.text_file_obj_2.name)]

```

Comment:
@alvacouch OK. Never mind, just realized the whole temp directory is deleted in tearDown(), so disregard my previous comment above.